types/index.esm.d.ts

```diff
@@ -1840,14 +1837,14 @@ export const BarElement: ChartComponent & {
 	new (cfg: any): BarElement;
 };
 
-export interface ElementOptions {
+export interface ElementOptionsByType {
 	arc: ArcOptions & ArcHoverOptions;
 	bar: BarOptions & BarHoverOptions;
 	line: LineOptions & LineHoverOptions;
 	point: PointOptions & PointHoverOptions;
 }
 export interface ElementChartOptions {
-	elements: ElementOptions;
+	elements: { [k in keyof ElementOptionsByType]: ElementOptionsByType[k]; };
```

Comment:
I think this is equivalent to `elements: Partial<ElementOptionsByType>`? If so, I find `Partial` more readable.